src/Workspaces/CSharp/Portable/LanguageServices/CSharpSyntaxFactsService.cs

```diff
@@ -462,7 +462,7 @@ public bool IsLiteral(SyntaxToken token)
 
         public bool IsStringLiteral(SyntaxToken token)
         {
-            return token.IsKind(SyntaxKind.StringLiteralToken);
+            return token.IsKind(SyntaxKind.StringLiteralToken, SyntaxKind.InterpolatedStringTextToken);
```

Comment:
I'm not sure whether IsStringLiteral should return true for InterpolatedStringTextTokens or whether we need another extension here to cover both. Opinions welcomed. Also, maybe some tests will fail and answer the question for me.